code/game/objects/random/random.dm

```diff
@@ -875,23 +875,32 @@ obj/random/obstruction/item_to_spawn()
 	desc = "LOADSAMONEY!"
 	icon = 'icons/obj/items.dmi'
 	icon_state = "spacecash1"
+	spawn_nothing_percentage = 75
 
 /obj/random/cash/item_to_spawn()
-	return pickweight(list(/obj/item/weapon/spacecash/ewallet/random/c200 = 8,
+	return pickweight(list(
+				/obj/item/weapon/spacecash/ewallet/random/c50 = 24,
+				/obj/item/weapon/spacecash/ewallet/random/c100 = 16,
+				/obj/item/weapon/spacecash/ewallet/random/c200 = 8,
 				/obj/item/weapon/spacecash/ewallet/random/c500 = 6,
-				/obj/item/weapon/spacecash/ewallet/random/c1000 = 4,
-				/obj/item/weapon/spacecash/ewallet/random/c5000 = 2,
-				/obj/item/weapon/spacecash/ewallet/random/c10000 = 1))
+				/obj/item/weapon/spacecash/ewallet/random/c1000 = 4))
 
 /obj/random/cash_poor
 	name = "lesser random currency"
 	desc = "LOADSAMONEY!"
 	icon = 'icons/obj/items.dmi'
 	icon_state = "spacecash1"
+	spawn_nothing_percentage = 50
 
 /obj/random/cash_poor/item_to_spawn()
-	return pickweight(list(/obj/item/weapon/spacecash/ewallet/random/c200 = 9,
-				/obj/item/weapon/spacecash/ewallet/random/c500 = 1))
+	return pickweight(list(
+				/obj/item/weapon/spacecash/ewallet/random/c1 = 96,
+				/obj/item/weapon/spacecash/ewallet/random/c5 = 48,
+				/obj/item/weapon/spacecash/ewallet/random/c10 = 24,
+				/obj/item/weapon/spacecash/ewallet/random/c25 = 16,
+				/obj/item/weapon/spacecash/ewallet/random/c50 = 8,
+				/obj/item/weapon/spacecash/ewallet/random/c100 = 4,
+				/obj/item/weapon/spacecash/ewallet/random/c200 = 2))
```

Comment:
This is only really used for crates. And they should ONLY be used as such. And these values are dumb. Literally given a more than 50% chance to get a penny out of a crate. Please just leave this subtype as is, a 90% chance to get 200 bucks really isn't that much. Intending to put more stuff in crates later, but for now it is what it is.
![image](https://user-images.githubusercontent.com/28861083/170452119-7e53603b-2108-40eb-a6d8-38fcdf9657f4.png)

Reply:
Honestly, I'm not finding much in this PR that was really given some serious concerns. Finding obscene amounts of money should be exceedingly rare, and finding lower amounts (you know, the grind to get more things -> keeping players occupied -> engaged -> player retention?) will serve you better.